softdev.bib

```diff
@@ -2757,3 +2757,43 @@ @misc{watson21assessing
     month = Sep,
     url = "https://www.capabilitieslimited.co.uk/pdfs/20210917-capltd-cheri-desktop-report-version1-FINAL.pdf"
 }
+
+@misc{servo,
+    title = {The Servo Parallel Browser Engine Project},
+    howpublished = {\href{https://servo.org}{https://servo.org}},
+    note = {Accessed: 2022-12-17},
+    year = {2012},
+    author = {Mozilla},
+}
+
+@misc{rustgc,
+    title = {rust-gc: simple tracing (mark and sweep) garbage collector for Rust},
+    howpublished = {\href{https://github.com/manishearth/rust-gc/}{https://github.com/manishearth/rust-gc/}},
+    note = {Accessed: 2022-12-17},
+    year = {2015},
+    author = {Manish Goregaokar},
+}
+
+@misc{shifgrethor,
+    title = {shifgrethor},
+    howpublished = {\href{https://github.com/withoutboats/shifgrethor/}{https://github.com/withoutboats/shifgrethor/}},
+    note = {Accessed: 2022-12-17},
+    year = {2018},
+    author = {withoutboats},
+}
+
+@misc{luster,
+    title = {luster - An experimental Lua VM implemented in pure Rust},
+    howpublished = {\href{https://github.com/kyren/luster/}{https://github.com/kyren/luster/}},
+    note = {Accessed: 2022-12-17},
+    year = {2018},
+    author = {kyren},
```

Comment:
I think her name is Catherine West?

Reply:
I also found this by googling kyren, but wasn't sure if it was right to use this instead as I saw no reference to it from the github profile.

Reply:
We should use real names unless there's a very good reason not to (I think the only reason would be: we can't find out that person's name).

Reply:
Fixed: 1971e4d7d9b293e9df794079506636667b83ebd7